src/content/en/2020/http2.md

```diff
@@ -10,7 +9,7 @@ analysts: [gregorywolf]
 translators: []
 dotjs_bio: Andrew works at <a href="https://www.cloudflare.com/">Cloudflare</a> helping to make the web faster and more secure. He spends his time deploying, measuring and improving new protocols and asset delivery to improve end-user web site performance.
 rmarx_bio: Robin is a web protocol and performance researcher at <a href="https://www.uhasselt.be/edm">Hasselt University, Belgium</a>. He has been working on getting QUIC and HTTP/3 ready to use by creating tools like <a href="https://github.com/quiclog">qlog and qvis</a>.
-#MikeBishop_bio: TODO
+MikeBishop_bio: Editor of HTTP/3 with @quicwg. Architect in Akamai's Foundry group.
```

Comment:
@MikeBishop I grabbed this from your GitHub profile, but please suggest a different bio if you'd prefer.

Reply:
In case @MikeBishop doesn't get back let's change the QUIC WG link since it's no longer on GitHub:

```suggestion
MikeBishop_bio: Editor of HTTP/3 with the [QUIC Working Group](https://quicwg.org/). Architect in [Akamai](https://www.akamai.com/)'s Foundry group.
```